Manager/BackupManager.php

```diff
@@ -62,13 +63,22 @@ public function execute()
             // Transfer with all clients
             $this->cm->upload($this->processor->getArchivePath());
 
-            $this->logger->info('[dizda-backup] Cleaning up after us.');
-            $this->processor->cleanUp();
         } catch (\Exception $e) {
-            // write log
+            // Write log
             $this->logger->critical('[dizda-backup] Unexpected exception.', array('exception' => $e));
 
             return false;
+        } finally {
```

Comment:
In fact, this is coherent.

But the master branch of this bundle currently support php up to 5.3.
I suppose that we could break many installations if we break the backward compatibility with the 5.3, what do you think?